learnify/mobile/assets/lang/en.json

```diff
@@ -49,5 +49,8 @@
   "events": "Events",
   "forum": "Forum",
   "create_chapter": "Create a Chapter",
-  "edit_chapter": "Edit the Chapter"
+  "edit_chapter": "Edit the Chapter",
+  "taken_courses": "Taken Courses",
+  "friend_courses": "Friends' Courses",
+  "recommended_courses": "Recommended Courses"
```

Comment:
We should change the "course" terms with "learning space" according to the customer's feedback. Also, there won't be chapter section anymore. We should also change related terms with "post".

Reply:
Done